### PR TITLE
JSON Trail

### PR DESCRIPTION
**Pull request by CashewTong**

### CODE REVIEW

File: models/solenoids/em_force_model.py [hunk 1]
```diff
@@ -143,4 +134,4 @@ def calc_force(area, mf):
 '''
 
 # if __name__ == "__main__":
-#   input_values()
+#   input_values()
```
Comment (hannahvsawiuk) [inline models/solenoids/em_force_model.py]: Add in new line at the end of the file

File: models/solenoids/awg_data.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        },
+
+        "28": {
+
+            "mm": "0.32004"
+            
+        }
+
+    }
+}
```
Comment (hannahvsawiuk) [inline models/solenoids/awg_data.json]: Add new line at the EOF

File: models/solenoids/GUIscratch.py [hunk 3]
```diff
@@ -1,10 +1,15 @@
+from matplotlib.font_manager import json_load
```
Comment (hannahvsawiuk) [inline models/solenoids/GUIscratch.py]: this was fixed. Remove

Comment (hannahvsawiuk) [reply models/solenoids/GUIscratch.py]: incorrect JSON package

**Comment by hannahvsawiuk** [comment]
@CashewTong rebase and resolve the merge conflicts pls. We can do this tonight

**Comment by CashewTong** [comment]
Already pushed to main

**Pull request closed without merging**